src/Server/src/GraphQL/GraphQLServiceCollectionExtensions.cs

```diff
@@ -112,7 +112,13 @@ private static IRequestExecutorBuilder AddTypes(this IRequestExecutorBuilder bui
                 .AddType<IdentityServerEventType>()
                 .AddType<ClientTemplateType>()
                 .AddType<UserClaimRuleType>()
-                .AddType<ClientTemplateSecretType>();
+                .AddType<ClientTemplateSecretType>()
+                .AddTypeExtension(new ObjectTypeExtension(t =>
```

Comment:
Wait what? this works?

Reply:
I guess i know why this works. But this would make use of a implementation detail.
It would be at least as valid to throw a schema exception here (in fact, i wonder if we should :D ) 

I have to look into this